sudoku.h

```diff
@@ -2,7 +2,8 @@
 class Sudoku
 {
 private:
-	int sudoku[9][9];
+	const int size = 9;
+	static int sudoku[size][size];
```

Comment:
Do not define sudoku member static as you can have more than one objects with different symbols.
Instead, you can make size both static and const